exotica_core/include/exotica_core/kinematic_tree.h

```diff
@@ -198,6 +198,13 @@ class KinematicTree : public Uncopyable
     /// @param end link name at which the chain ends
     /// @return list joints between begin and end
     std::vector<std::string> GetKinematicChain(const std::string& begin, const std::string& end);
+
+    /// @brief GetKinematicChainLinks get the links between begin and end of kinematic chain
+    /// @param begin link name from which the chain starts
+    /// @param end link name at which the chain ends
+    /// @return list link between begin and end
+    std::vector<std::string> GetKinematicChainLinks(const std::string& begin, const std::string& end);
```

Comment:
`std::vector<std::string> GetKinematicChainLinks(const std::string& begin, const std::string& end) const;`

Reply:
I made both methods (`GetKinematicChain` and `GetKinematicChainLinks`) const.